site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -765,7 +765,17 @@ components:
                type: array
                items:
                   type: string
-               description: The device IDs of the gateways this device is assigned to.
+               description: The device IDs of the gateways that are registered to act on behalf of this device. Note that "via" and "memberOf" must not be set at the same time.
+            "viaGroups":
+               type: array
+               items:
+                  type: string
+               description: The IDs of the gateway groups that are registered to act on behalf of this device. Note that "viaGroups" and "memberOf" must not be set at the same time.
+            "memberOf":
+               type: array
+               items:
+                  type: string
+               description: The IDs of the gateway groups in which this device is a member. Note that "via" and "memberOf" must not be set at the same time. The same applies for "viaGroups" and "memberOf" which must not be set at the same time too. The reason is that Eclipse Hono does not support groups of gateway groups.
```

Comment:
can you please bump the version of the spec to `1.1.0`

Reply:
Done, I hope I understood you correctly and adapted the right version parameter.

Reply:
you did